stl/inc/functional

```diff
@@ -1818,7 +1818,7 @@ public:
         return *this;
     }
 
-    move_only_function& operator=(move_only_function&& _Other) {
+    move_only_function& operator=(move_only_function&& _Other) noexcept {

```

Comment:
Only the change in this line is reasonable to me.

However, it seems that some people in LWG/LEWG want to keep this move assignment operator non-noexcept, see https://github.com/microsoft/STL/issues/2278#issuecomment-943360927.

Reply:
If anything, this should have `noexcept(true)` with a comment explaining why we don't want to strengthen it so we don't have to keep rediscovering why.